addon/components/validated-input/template.hbs

```diff
@@ -31,7 +35,19 @@
         {{option.label}}</label>
     </div>
   {{/each}}
-
+{{else if (eq type "checkbox")}}
+  <div class="{{config.css.checkbox}}">
+    <label>
+      {{one-way-checkbox
+          checked
+          name       = name
+          value      = value
```

Comment:
This should probably be `checked = (get model name)`, since there's no `value` attribute on one-way-checkbox